src/layup/orbitfit.py

```diff
@@ -103,6 +115,169 @@ def _use_star_astrometry(data):
     return data
 
 
+def _split_by_index(input_list, indices):
+    result = []
```

Comment:
do we docstrings or just quick descriptions of what this function (and the following ones) do?